Tasks/google-play-release/Tests/L0.ts

```diff
@@ -70,6 +70,16 @@ describe('L0 Suite google-play-release', function () {
         done();
     });
 
+    it('test pass on Obb file not found', (done) => {
+        const testFile = path.join(__dirname, 'L0ObbFileNotFound.js');
+        const testRunner = new ttm.MockTestRunner(testFile);
+
+        testRunner.run();
+
+        assert(testRunner.succeeded, 'task should have succeeded' );
```

Comment:
This is pretty good.  It's roughly 98% there.  You should assert that you expect a message to be printed.  This means that you may need add a line to print in the console that it's 'using obb file' or something.

Reply:
You mean if the box is checked and there is no obb, I should print a message on the console saying like "Skipping upload"?

Reply:
I there there should be one that tries to upload.  You can require a file using something like [this](https://github.com/microsoft/google-play-vsts-extension/blob/6cb76f6e79c779af0d379f2b021e1551d706344c/Tasks/google-play-release/Tests/L0FoundDeobfuscationFile.ts#L25)

Reply:
In the example you mentioned, there is a field in the UI ('mappingFilePath') in which we can set the file path, but in this case it just picks up from the parent directory of the apk right?, so how do I approach this?

Reply:
You need to mock the modules that you use.  For instance, I see that you're using `fs.readFileSync`.  [This](https://github.com/microsoft/google-play-vsts-extension/blob/6cb76f6e79c779af0d379f2b021e1551d706344c/Tasks/google-play-release/Tests/L0AttachMetadata.ts#L60) also uses `fs.readFileSync`, so your 'answers' will need to include what you would expect to be read in that.